torch/_torch_docs.py

```diff
@@ -3035,8 +3035,10 @@ def merge_dicts(*dicts):
 Creates a :class:`Tensor` from a :class:`numpy.ndarray`.
 
 The returned tensor and :attr:`ndarray` share the same memory. Modifications to
-the tensor will be reflected in the :attr:`ndarray` and vice versa. The returned
-tensor is not resizable.
+the tensor will be reflected in the :attr:`ndarray` and vice versa. 
+Writing to a tensor created from a read-only NumPy array is not supported and 
```

Comment:
This should be a warning, see here:

https://github.com/pytorch/pytorch/blob/5c3788d5d76f64f6708e0b79f40b1cf45276625a/torch/_torch_docs.py#L339

And it should appear just before the example.